internal/server/recon/golden/recognize_places_test.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Google LLC
```

Comment:
nit: its a bit confusing to have two files with the same name? maybe call this integration test?

Reply:
The golden test file name usually corresponds to the RPC method name - so I changed the name of the code files from recognize_places.go to recognize.go, because we might need to recognize other things in the future, and those code can all go there.